base_layer/core/src/validation/helpers.rs

```diff
@@ -156,6 +156,16 @@ pub fn is_stxo<T: BlockchainBackend>(db: &T, hash: HashOutput) -> Result<bool, V
     }
 }
 
+pub fn is_utxo_in_mmr<T: BlockchainBackend>(db: &T, hash: HashOutput) -> Result<bool, ValidationError> {
+    // Check if the UTXO MMR contains the specified deleted UTXO hash, the backend stxo_db is not used for this task as
+    // archival nodes and pruning nodes might have different STXOs in their stxo_db as horizon state STXOs are
+    // discarded by pruned nodes.
```

Comment:
This comment does not seem to reflect what happens below; it is an exact copy of the comment for `pub fn is_stxo<T: BlockchainBackend>(db: &T, hash: HashOutput) -> Result<bool, ValidationError> {`